.devops/kubernetes/helmCharts/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
the folder can be only helm?

because helmChart is redundant

Reply:
ok